package.json

```diff
@@ -55,7 +55,10 @@
     "webpack": "^1.13.0"
   },
   "peerDependencies": {
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1"
+    "react": "^15.0.1 || ^16.0.0 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^15.0.1 || ^16.0.0 || ^17.0.0 || ^18.0.0"
+  },
+  "dependencies": {
+    "create-react-class": "^15.7.0"
```

Comment:
This seems to be used in the tests only, right? Can it become a `devDependency`?